### ISSUE TITLE
[Admin] 403 Access denied after updating admin user

### ISSUE THREAD

**Issue by michalmarcinkowski** [Sylius/Sylius#5227]
The following scenario describes the bug:
```
Given I am logged in as an administrator
When I want to edit this user
And I deselect "Customer can login to the store"
But then select it again
And I save my changes
Then I get 403 access denied exception
And I am not an administrator any more :(
```

**Comment by Arminek** [comment]
I have reproduced this bug, but in current master version i didn't get the access denied exception.

**Comment by michalmarcinkowski** [comment]
The following scenario still occurs:
```
Given I am logged in as an administrator
When I change my password to "abc"
And I logout
And I login to the admin panel with password "abc"
Then I get 403 access denied exception
And I am not an administrator any more :(
```

**Comment by Sicaine** [comment]
Same here. It happens as long as you save the user. Doesn't matter what you change.

**Comment by bwach** [comment]
I couldn't reproduce it in current master

**Comment by michalmarcinkowski** [comment]
Fixed in #5466
